test_runner/flank.yml

```diff
@@ -107,6 +107,11 @@ flank:
   ## Google cloud storage path to store the JUnit XML results from the last run.
   # smart-flank-gcs-path: gs://tmp_flank/flank/test_app_android.xml
 
+  ## Whether the JUnit XML results should be uploaded for Smart Flank.
+  ## This can be disabled to prevent new results from overriding previous results.
+  ## Default: true
+  # smart-flank-upload-enabled: true
+
   ## Disables sharding. Useful for parameterized tests.
   # disable-sharding: false
 
```

Comment:
I think for consistency with `disable-sharding`, I'd like the flag renamed to:

`smart-flank-disable-upload`

then in the code we can check `args.smartFlankDisableUpload` instead of `!args.smartFlankUploadEnabled`.

The description can be written like this:
> Disables smart flank JUnit XML uploading. Useful for preventing timing data from being updated.

I'm happy to make both changes if you'd like.

Reply:
sure thing, I'll do it

Reply:
Perfect, thanks. After that I'll merge the PR.